Adyen/Model/Checkout/DefaultPaymentMethodDetails.cs

```diff
@@ -102,7 +102,9 @@ public override string ToString()
             sb.Append("  EncryptedExpiryMonth: ").Append(EncryptedExpiryMonth).Append("\n");
             sb.Append("  EncryptedExpiryYear: ").Append(EncryptedExpiryYear).Append("\n");
             sb.Append("  EncryptedSecurityCode: ").Append(EncryptedSecurityCode).Append("\n");
+#pragma warning disable 618
             sb.Append("  RecurringDetailReference: ").Append(RecurringDetailReference).Append("\n");
+#pragma warning restore 618
```

Comment:
Would be nice to add a comment of what's being done here

Reply:
There are some fields that are obsolete and raise warnings if are used. We are still using them for backward compatibility as removing them will break implementations. I decided to suppress the warning as we know the cause.    [Here ](https://stackoverflow.com/questions/968293/c-sharp-selectively-suppress-custom-obsolete-warnings) there is a guide how to suppress warnings in c#